plus/build.gradle.kts

```diff
@@ -25,9 +25,11 @@ dependencies {
     implementation("com.teradata.tpcds:tpcds")
     implementation("io.prestosql.tpch:tpch")
     implementation("net.hydromatic:chinook-data-hsqldb")
+    implementation("net.hydromatic:sql-logic-test:0.2")
```

Comment:
Should this rather be a `testImplementation` dependency?

Reply:
Also probably the version ("0.2") should not be controlled here. That's why we have the gradle.properties file.